timeout.c

```diff
@@ -122,17 +122,25 @@
 #define WHEEL_MASK (WHEEL_LEN - 1)
 #define TIMEOUT_MAX ((TIMEOUT_C(1) << (WHEEL_BIT * WHEEL_NUM)) - 1)
 
+#include "bitops.c"
+
 #if WHEEL_BIT == 6
+#define ctz(n) ctz64(n)
+#define clz(n) clz64(n)
+#define fls(n) ((int)(64 - clz64(n)))
+#else
+#define ctz(n) ctz32(n)
+#define clz(n) clz32(n)
+#define fls(n) ((int)(32 - clz32(n)))
+#endif
 
+#if WHEEL_BIT == 6
```

Comment:
Yes, but note that the #else on 131 covers the WHEEL_BIT == 5, 4, and 3 cases, whereas they are handled separately below